cmlutils/project_entrypoint.py

```diff
@@ -156,8 +157,17 @@ def project_export_cmd(project_name):
             project_slug=project_slug,
             owner_type=owner_type,
         )
+        start_time = time.time()
         pexport.transfer_project_files(log_filedir=log_filedir)
-        pexport.dump_project_and_related_metadata()
+        exported_data = pexport.dump_project_and_related_metadata()
+        print("\033[32m✔ Export of Project {} Successful \033[0m".format(project_name))
```

Comment:
Instead of using print statements, use logging.info(). We should preserve these logs in migration.log file as well.

Reply:
So if you remember in one of our sync manas suggested we should show a summary of what is migrated and a status